composeApp/src/androidMain/kotlin/it/fast4x/rimusic/ui/components/themed/PlaylistsItemMenu.kt

```diff
@@ -566,6 +570,28 @@ fun PlaylistsItemMenu(
                             }
                         )
                     }
+
```

Comment:
Same thing.

Even better, make a separate function handles this particular thing

Reply:
I understand that they are kind of repetitive. But, Since all the other menu items are listed this way, putting only these 2 items in a separate function and then calling them here will lead to more confusion is what I feel.